README.md

```diff
@@ -257,6 +257,52 @@ codacy-analysis-cli analyse \
 
 > In alternative to setting `--api-token` you can define CODACY_API_TOKEN in the environment.
 
+## GitHub Action
+
+The analysis CLI is available to be executed as a GitHub Action. The following is an example of a workflow
+using the CLI as an action, to analyse each commit and pull request, with your Codacy configuration.
+
+```yaml
+name: codacy-analysis-cli
+
+on: ["push"]
+
+jobs:
+  codacy-analysis-cli:
+    runs-on: ubuntu-latest
+    name: codacy-analysis-cli
+    steps:
+      - uses: actions/checkout@master
+      - name: Run codacy-analysis-cli
+        uses: codacy/codacy-analysis-cli@master
+        with:
+          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+```
+
+Running the action with the default configurations will:
+
+- Analyse the current commit or pull request, using your Codacy configurations, running all supported
```

Comment:
What do you think about saying it analyses the files in the current directory or the specified directory instead? Talking about commits and PRs at this level seems too complicated.

Reply:
This documentation is in the context of using a GH action. I think commits and PRs are very common things for people using GH actions.

Reply:
Anyway, I think I will need @pauloribeiro-codacy help to improve the documentation of the CLI in general and of the action in particular.

Reply:
I've simplified the example and the description, so I will close this discussion.